float64[] horizontal_velocity
float64 yaw_rate
float64 height
float64 pitch
float64 roll
int8 activation

bool hop_event
bool trot_event
bool activate_event